doc/reference/config-options-launch.rst

```diff
@@ -0,0 +1,103 @@
+.. _launch-config-options:
+
+Launch Configuration Options
+##############################
+
+As explained in :ref:`acrn_configuration_tool`, launch configuration files
+define post-launched User VM settings. This document describes these option settings.
+
+``uos``:
+  Specify the User VM with its relative ID to Service VM by the ``id``
+  attribute.
+
+``uos_type``:
+  Specify the User VM type, such as ``CLEARLINUX``, ``ANDROID``, ``ALIOS``,
+  ``PREEMPT-RT LINUX``, ``GENERIC LINUX``, ``WINDOWS``, ``YOCTO``, ``UBUNTU``,
+  ``ZEPHYR`` or ``VXWORKS``.
+
+``rtos_type``:
+  Specify the User VM Real-time capability: Soft RT, Hard RT, or none of them.
+
+``mem_size``:
+  Specify the User VM memory size in megabytes.
+
+``gvt_args``:
+  GVT arguments for the VM. Set it to ``gvtd`` for GVT-d, otherwise it's
+  for GVT-g arguments.  The GVT-g input format:
+  ``low_gm_size high_gm_size fence_sz``,
+  The recommendation is ``64 448 8``.  Leave it blank to disable the GVT.
+
+``vbootloader``:
+  Virtual bootloader type; currently only supports OVMF.
+
+``vuart0``:
+  Specify whether the device model emulates the vUART0(vCOM1); refer to
+  :ref:`vuart_config` for details.  If set to ``Enable``, the vUART0 is
+  emulated by the device model; if set to ``Disable``, the vUART0 is
+  emulated by the hypervisor if it is configured in the scenario XML.
+
+``poweroff_channel``:
+  Specify whether the User VM power off channel is through the IOC,
+  power button, or vUART.
+
+``allow_trigger_s5``:
+  Allow VM to trigger s5 shutdown flow, this flag works with
+  ``poweroff_channel``
+  ``vuart1(pty)`` and ``vuart1(tty)`` only.
+
+``enable_ptm``:
+  Enable the Precision Timing Measurement (PTM) feature.
+
+``usb_xhci``:
+  USB xHCI mediator configuration. Input format:
+  ``bus#-port#[:bus#-port#: ...]``, e.g.: ``1-2:2-4``.
+  Refer to :ref:`usb_virtualization` for details.
+
+``shm_regions``:
+  List of shared memory regions for inter-VM communication.
+
+``shm_region`` (a child node of ``shm_regions``):
+  configure the shared memory regions for current VM, input format:
+  ``hv:/<;shm name>; (or dm:/<shm_name>;), <;shm size in MB>;``. Refer
```

Comment:
> 

That's not quite right either (it leaves off the VM IDs that can share this region)...

Reply:
I _think_ that it's right as it is, because the VM ID is known because you're in a specific VM section. It's in the scenario file that the VM IDs need to be listed. @NanlinXie @junjiemao1 can you confirm?

Reply:
@NanlinXie Could you please forward this question to the owner of ivshmem for a clarification?

Reply:
VM IDs are not needed for this config, only need the name of the shared memory region and its size(in MB). @gvancuts ’s comment is correct.